src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_item.html.twig

```diff
@@ -1,5 +1,5 @@
 <div class="sf-toolbar-block sf-toolbar-block-{{ name }} sf-toolbar-status-{{ status|default('normal') }} {{ additional_classes|default('') }}" {{ block_attrs|default('')|raw }}>
-    {% if link is not defined or link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
+    {% if link is not defined or link %}<a href="{{ link ? link : path('_profiler', { token: token, panel: name }) }}">{% endif %}
```

Comment:
this change is wrong, because it changes the behavior of ` link`  from being a boolean (should the item have a link or no) to being the URL of the link. So this is a BC break for bundles using the item (and it does not use a variable passed from the controller either, as that template is not rendered from the controller directly).

Reply:
The link is generated in the ProfilerController (149) and passed via the toolbar.html.twig and passed in block('toolbar') which ends with
`{{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: profiler_url }) }}`
So that's where I got the pre-generated path from. If it needed to be a boolean, the previous code was already incorrect.